docs: fix ISO language codes link to Sphinx language option link (#354)
pyOpenSci/python-package-guide: TRANSLATING.md

```

```{note}

> You can find a list of the two-letter IETF language tag [here](https://en.wikipedia.org/wiki/IETF_language_tag).

Author: ```suggestion
You can find a list of the two-letter sphinx language option [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language).
```

> You can find a list of the two-letter IETF language tag [here](https://en.m.wikipedia.org/wiki/IETF_language_tag).

Author: ```suggestion
You can find a list of the two-letter IETF language tag [here](https://en.wikipedia.org/wiki/IETF_language_tag).
```

> You can find a list of the two-letter sphinx language option [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language).

Author: ```suggestion
You can find a list of the two-letter Sphinx language option [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language).
```
